cmd/ko/README.md

```diff
@@ -0,0 +1,146 @@
+# `ko`
+
+## What is `ko`?
+
+`ko` is an **experimental** CLI to support rapid development of Go containers
+with Kubernetes.
+
+## Installation
+
+`ko` can be installed via:
+
+```shell
+go install github.com/google/go-containerregistry/cmd/ko
+```
+
+## The `ko` Model
+
+`ko` is built around a very simple extension to Go's model for expressing
+dependencies using import paths. In Go, dependencies are expressed via
+blocks like:
+
+```go
+import (
+    "github.com/google/go-containerregistry/authn"
+    "github.com/google/go-containerregistry/name"
+)
+```
+
+Similarly (as you can see above), Go binaries can be references via import
+paths like `github.com/google/go-containerregistry/cmd/ko`.
+
+With `ko`, we simply replace image references within Kubernetes yaml
```

Comment:
This sentence is kind of hard to understand, and I think you've buried the lede a bit.

The Big Idea™ is hidden in here, i.e.:
> if I use `ko`, I don't have to worry about building and pushing containers anymore. I can just pretend kubernetes knows how to deploy my go binaries, and the container becomes an implementation detail.

I don't think that's immediately obvious unless you spell it out for attention-deficient folks like me. Splitting out the motivation and the implementation could make this sentence easier to swallow, too.

Reply:
I've rephrased this to play up containers as an implementation detail that fades away.